library/src/main/java/com/amar/library/ui/StickyScrollView.java

```diff
@@ -56,9 +53,12 @@ public void onGlobalLayout() {
     @Override
     protected void onLayout(boolean changed, int l, int t, int r, int b) {
         super.onLayout(changed, l, t, r, b);
-        if(stickyFooterView != null && !changed) {
-            stickyFooterView.getLocationInWindow(updatedFooterLocation);
-            mStickyScrollPresenter.recomputeFooterLocation(getRelativeTop(stickyFooterView), updatedFooterLocation[1]);
+        if (stickyFooterView != null && !changed) {
+            mStickyScrollPresenter.recomputeFooterLocation(getRelativeTop(stickyFooterView));
+        }
+
+        if (stickyHeaderView != null) {
+            mStickyScrollPresenter.recomputeHeaderLocation(stickyHeaderView.getTop());
```

Comment:
Can you please do just this in this PR and create a separate PR for other things if possible?

Reply:
will do in a day or so 👍